### DISCUSSION TITLE
How to implement weather

### DISCUSSION THREAD

**Discussion by Drizzt893** [haven1433/HexManiacAdvance#171] [Q&A]
Modding Leaf Green and looking to add weather to the elite four battles, not via an attack or ability, but automatically at the start of battle like when you're near Kyogre in gen 3.  How do we implement this?  Can't seem to find anything about weather at all for this program.  Thanks!

**Comment by haven1433** [comment]
Weather stuff is usually in the map header panel. But I don't think weather was implemented in FireRed / LeafGreen.
